src/language/tekster_en.ts

```diff
@@ -0,0 +1,105 @@
+  'dokumenter.seAlle': 'See all documents',
+  'dokumenter.henter': 'Fetching documents',
+  'dokumenter.galt': 'Something went wrong while fetching your documents.',
+
+  'søke.tittel': 'Do you want to apply?',
+
+  'overgangsstønad.tittel': 'Transitional benefit',
+  'overgangsstønad.brødtekst':
+    'Ensures you income for up to 3 years when you have at least 60 per cent of the daily care for children under 8 years of age.',
+  'overgangsstønad.lenkeTekst': 'Read more about transitional benefit',
+  'overgangsstønad.knapp': 'Apply for transitional benefit',
+
+  'barnetilsyn.tittel': 'Childcare benefit',
+  'barnetilsyn.brødtekst':
+    'Covers part of the cost for kindergarten, after-school program (SFO) or daycare when you have sole care of a child and are working.',
+  'barnetilsyn.lenkeTekst': 'Read more about childcare benefit',
+  'barnetilsyn.knapp': 'Apply for childcare benefit',
+
+  'skolepenger.tittel': 'Support for school fees',
+  'skolepenger.brødtekst':
+    'Covers expenses related to tuition fees, semester fees and examination fees for single parents when you are studying and are alone with children.',
+  'skolepenger.lenkeTekst': 'Read more about support for school fees',
+  'skolepenger.knapp': 'Apply for support for school fees',
+
+  'snarveier.tittel': 'Other shortcuts',
+  'snarveier.saksbehandlingstider': 'Case processing times',
+  'snarveier.skrivTilOss': 'Write to us',
+  'snarveier.utbetalingsoversikt': 'Payment overview',
+  'snarveier.utvidetBarnetrygd': 'Extended child benefit',
+
+  'tekst.seDetaljer': 'Click to see details',
+  'vedlegg.tittel': 'Attachments',
+
+  'journalpost.deg': 'Sent by you: [0]',
+  'journalpost.nav': 'Sent from NAV: [0]',
+
+  'dataViewer.henter': 'Fetching your benefits',
+  'dataViewer.galt': 'Something went wrong while fetching your benefits.',
+
+  'stønad.fraTil': 'You have benefits from [0] to [1]',
+  'stønad.tilOgMed': 'You have benefits until [0]',
+
+  'utbetaling.tittel': 'Your payments',
+  'utbetaling.brødtekst': 'Here you can check what you have received from NAV.',
+
+  'dokumentoversikt.breadcrumb': 'Document overview',
+  'overgangsstønad.breadcrumb': 'Your transitional benefit',
+  'barnetilsyn.breadcrumb': 'Your childcare benefit',
+  'skolepenger.breadcrumb': 'Your school fee benefit',
+
+  'overgangsstønad.beskrivelse': 'Here are your decision documents for transitional benefit.',
+  'barnetilsyn.beskrivelse': 'Here are your decision documents for childcare benefit.',
+  'skolepenger.beskrivelse': 'Here are your decision documents for school fee benefit.',
+
+  'overgangsstønad.brødtekst.util':
+    'The table shows your periods with transitional benefit, how much you have received or will receive per month, and the income we have used to calculate your benefit. To see how we have calculated your benefit, you must read your decision. You can find your decisions in the document overview further down the page.',
```

Comment:
kankskje "please refer to your decision letter" i stedet for "you must read .." ?